src/shared/ui/logo/logo.tsx

```diff
@@ -0,0 +1,32 @@
+import { HTMLAttributes } from 'react';
+import { VariantProps, cva } from 'class-variance-authority';
+import classNames from 'classnames/bind';
+import logo from '@/shared/assets/svg/rss-logo.svg';
+import Image from '@/shared/ui/image';
+
+import styles from './logo.module.scss';
+
+type LogoProps = HTMLAttributes<HTMLElement> & VariantProps<typeof logoVariants>;
+
+export const cx = classNames.bind(styles);
+
+const logoVariants = cva(cx('logo'), {
+  variants: {
+    type: {
+      default: cx(''),
+      'with-border': cx('with-border'),
+    },
+  },
+  defaultVariants: { type: 'default' },
+});
+
+export const Logo = ({ type, ...props }: LogoProps) => {
```

Comment:
Why do we need `props` here?

Reply:
I described them by analogy with other components so that the code would be identical. I think they are not needed here, I deleted them

Reply:
If you decided to remove the `props` from the component, please make sure to update the interface accordingly, because I can still pass props, but they won't have any effect.

Reply:
fixed